cloudconfig/cloudinit/cloudinit_ubuntu.go

```diff
@@ -279,6 +280,23 @@ func (cfg *ubuntuCloudConfig) addRequiredPackages() {
 	}
 }
 
+var waitSnapSeeded = `
+n=1
```

Comment:
I'm guessing AddRunCmd inserts this into a script/runs these commands in a shell? Just taking note of no #!

Reply:
yeah, that's how it works, we use a similar construct elsewhere too